src/features/applications/components/ApplicationReviewActions/ApproveButton.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from "~/components/ui/Button";
+
+type Props = {
+  disabled?: boolean;
+  isLoading?: boolean;
+  onClick?: () => void;
+};
+
+export function ApproveButton({ disabled, isLoading, onClick }: Props) {
+  return (
+    <Button
+      size="sm"
+      variant="primary"
+      disabled={disabled}
+      isLoading={isLoading}
+      onClick={onClick}
+    >
+      Approve
+    </Button>
+  );
```

Comment:
Would be great if we could avoid these kinds of components that just delegates a few props (size and variant). 

It might be better to add the hook to this component so it handles the loading and error states (for example displaying a toast). Keep in mind that the loading state wouldnt be accessible outside of the component.

Reply:
I agree, I am waiting for applying this changes in ApplicationItem and create a Button for all ApplicationReview actions.